### PR TITLE
Ca 61 change name and create account page

### PR DESCRIPTION
**Pull request by DawsonDease**
Created the account page and the user can now successfully change their name

### CODE REVIEW

File: screens/AccountScreen.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import { Button, Appbar } from "react-native-paper";
+import AccountModal from "../components/AccountModal";
+
+const AccountScreen = (props) => {
+  //determines what component the modal will render when a button is clicked
+  const [state, setState] = React.useState("");
```
Comment (katiemcgoff) [inline screens/AccountScreen.js]: This should be a more specific variable name than "state", this is too generic and confused with the general concept of state, useState(), etc. Something like "action" or "element" would be good here.

File: components/AccountModal.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Modal, Portal, Card } from "react-native-paper";
+import ChangeName from "../components/ChangeName";
+import ChangeEmail from "../components/ChangeEmail";
+import ChangePassword from "../components/ChangePassword";
+//OpenModal opens the modal and takes in the visibility which either show the modal or hides it
+//It also takes in the state which the button passes to it and will render the correct component
```
Comment (katiemcgoff) [inline components/AccountModal.js:7]: Excellent summary comment.

**Review by katiemcgoff — CHANGES_REQUESTED** [review]
Excellent work. One change in the code requested, and we are missing the updated App.tests.js to match the changes to App.js.

**Comment by katiemcgoff** [comment]
Oh and I think we're missing the ChangeName test file as well.

**Review by katiemcgoff — APPROVED** [review]

**Review by JaycobDuffel — APPROVED** [review]

**Pull request merged**